libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -1508,7 +1522,7 @@ GCS_MAVLINK::update_receive(uint32_t max_time_us)
 
     // send a timesync message every 10 seconds; this is for data
     // collection purposes
-    if (tnow - _timesync_request.last_sent_ms > _timesync_request.interval_ms && !is_private()) {
+    if (tnow - _timesync_request.last_sent_ms > _timesync_request.interval_ms && !is_private() && !is_high_latency_link) {
```

Comment:
Instead of doing this we should set the interval to zero.

That will require setting the interval in the first place - same place we set the heartbeat interval :-)

This will allow people to quieten these outgoing timesync messages in other circumstances, which will be nice.

Reply:
Turns out the ``!is_private()`` takes care of that anyway, as I've flagged the HL Channel as private. So removed